Assets/LeapMotion/Modules/GraphicRenderer/Scripts/LeapGraphicRenderer.cs

```diff
@@ -179,6 +179,11 @@ public partial class LeapGraphicRenderer : MonoBehaviour, ISerializationCallback
       }
     }
 
+    private void Update() {
+      // Validate the attached space to support it changing at runtime.
+      validateSpaceComponent();
```

Comment:
I'd recommend doing the validation inside of LateUpdate right before the GraphicRenderer does all the rest of it's work.  That way the user can't destroy or switch spaces between Update and potentially mess things up.

Reply:
Oh! Cool. Will make that change as well